pkg/universe.dagger.io/python/test/test.cue

```diff
@@ -10,50 +10,60 @@ import (
 dagger.#Plan & {
 	actions: test: {
 
-		// Run with a custom path to python
-		customPath: {
+		"python.cue": {
+			// Run with a custom path to python
+			customPath: {
 
-			// python:3.10-alpine has both python3 and python in the PATH
-			_image: docker.#Pull & {
-				source: "python:3.10-alpine"
-			}
-			run: python.#Run & {
-				input: _image.output
-				command: name:    "python"
-				script: contents: #"print("Hello, world!")"#
+				// python:3.10-alpine has both python3 and python in the PATH
+				_image: docker.#Pull & {
+					source: "python:3.10-alpine"
+				}
+				run: python.#Run & {
+					input: _image.output
+					command: name:    "python3"
+					script: contents: #"print("Hello, world!")"#
+				}
+				// This needs no output test because it is only testing that the command runs
 			}
-			// This needs no output test because it is only testing that the command runs
-		}
 
-		// Run a script from source directory + filename
-		runFile: {
+			// Run a script from source directory + filename
+			runFile: {
 
-			dir:   _load.output
-			_load: core.#Source & {
-				path: "./data"
-				include: ["*.py"]
+				dir:   _load.output
+				_load: core.#Source & {
+					path: "./data"
+					include: ["*.py"]
+				}
+
+				run: python.#Run & {
+					export: files: "/out.txt": _
+					script: {
+						directory: dir
+						filename:  "helloworld.py"
+					}
+				}
+				output: run.export.files."/out.txt" & "Hello, world\n"
 			}
 
-			run: python.#Run & {
-				export: files: "/out.txt": _
-				script: {
-					directory: dir
-					filename:  "helloworld.py"
+			// Run a script from string
+			runString: {
+				run: python.#Run & {
+					export: files: "/output.txt": _
+					script: contents: #"""
+						with open("output.txt", 'w') as f:
+							f.write("Hello, inlined world!\n")
+						"""#
 				}
+				output: run.export.files."/output.txt" & "Hello, inlined world!\n"
 			}
-			output: run.export.files."/out.txt" & "Hello, world\n"
 		}
 
-		// Run a script from string
-		runString: {
-			run: python.#Run & {
-				export: files: "/output.txt": _
-				script: contents: #"""
-					with open("output.txt", 'w') as f:
-					    f.write("Hello, inlined world!\n")
-					"""#
-			}
-			output: run.export.files."/output.txt" & "Hello, inlined world!\n"
+		"image.cue": version: {
+			[tag=string]:  python.#Image & {version: tag}
+			"3.10":        _
+			"3.10-alpine": _
+			"3.9":         _
+			"3.6":         _
```

Comment:
Without the alpine default, `3.X` images are bigger. Just use one custom alpine test otherwise it'll strain the CI runner more than necessary 🙂